torch/_dynamo/variables/torch.py

```diff
@@ -307,9 +310,24 @@ def call_function(
             return NullContextVariable(**options)
         elif self.value is torch.autograd._profiler_enabled:
             unimplemented("torch.autograd._profiler_enabled not supported yet")
+        elif self.unsupported_scalar_return() and not config.dynamic_shapes:
+            # wrap_fx_proxy can only support certain function types; functions that return scalars
+            # are not supported when dynamic shapes are not enabled.
```

Comment:
Why should this depend on dynamic shapes?  Dynamo specializes on dtype/device even in dynamic shape mode.

Reply:
sorry, meant to remove this